components/cap-content-router.js

```diff
@@ -0,0 +1,36 @@
+import { html, LitElement } from "../lib/lit.js";
+
+import "./cap-volume.js";
+import "./cap-case.js";
+import "./cap-jurisdictions.js";
+import "./cap-reporter.js";
+
+export default class CapContentRouter extends LitElement {
+	render() {
+		const searchParams = new URLSearchParams(window.location.search);
+		const reporter = searchParams.get("reporter");
+		const volume = searchParams.get("volume");
+		const caseName = searchParams.get("case");
+
+		if (caseName === null && volume === null && reporter === null) {
```

Comment:
Since we `return` in every branch here, we might be able to simplify this conditional quite a bit. 

Maybe something like:
```javascript
if (!!volume && !!caseName && !!reporter) {
	return html`<cap-case
		reporter=${reporter}
		volume=${volume}
		case=${caseName}
	></cap-case>`;	
}

if (!!volume && !!reporter) {
	return html`<cap-volume
		reporter=${reporter}
		volume=${volume}
	></cap-volume>`;
}

if (!!reporter) {
	return html`<cap-reporter reporter=${reporter}></cap-reporter>`;
}

return html`<cap-jurisdictions></cap-jurisdictions>`;
```

Reply:
I definitely think the example here improves the readability of the logic — someone only needs to understand what each `if` encapsulates.